app/src/main/res/layout/fragment_pin_view.xml

```diff
@@ -156,5 +167,70 @@
             android:layout_marginStart="20dp"
             app:layout_constraintStart_toEndOf="@+id/pin_found_count"
             app:layout_constraintTop_toBottomOf="@+id/pin_caption" />
+
+        <ImageView
+            android:id="@+id/add_comment_button"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:adjustViewBounds="false"
+            android:background="@drawable/ic_circle"
+            android:backgroundTint="?attr/colorSecondary"
+            android:clickable="true"
+            android:focusable="true"
+            android:foreground="@drawable/ic_add_comment"
+            android:foregroundTint="?attr/colorOnSecondary"
+            android:scaleType="fitCenter"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintTop_toTopOf="parent" />
+
     </androidx.constraintlayout.widget.ConstraintLayout>
+
+            <ImageView
+                android:id="@+id/send_comment_button"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:adjustViewBounds="false"
```

Comment:
What does this do?

Reply:
This was leftover from a Stack Overflow copy lol. Deleted